wagtailio/areweheadlessyet/models.py

```diff
@@ -1,12 +1,36 @@
 from django.db import models
-from wagtail.admin.edit_handlers import FieldPanel, MultiFieldPanel, StreamFieldPanel
+from modelcluster.fields import ParentalKey
+from wagtail.admin.edit_handlers import (
+    FieldPanel,
+    InlinePanel,
+    MultiFieldPanel,
+    PageChooserPanel,
+    StreamFieldPanel,
+)
 from wagtail.core.fields import RichTextField, StreamField
-from wagtail.core.models import Page
+from wagtail.core.models import Orderable, Page
 
 from wagtailio.areweheadlessyet.blocks import HomePageBlock
 from wagtailio.utils.models import CrossPageMixin, SocialMediaMixin
 
 
+class AreWeHeadlessYetNews(Orderable):
+    page = ParentalKey(
+        "areweheadlessyet.AreWeHeadlessYetHomePage",
+        on_delete=models.CASCADE,
+        related_name="news",
+    )
+    blog_post = models.ForeignKey(
+        "blog.BlogPage",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
```

Comment:
Suggestion: I think in this case we actually want `models.CASCADE`. Otherwise we're left with a `AreWeHeadlessYetNews` without a related page, which could cause a lot of issues. This way, the `AreWeHeadlessYetNews` is just deleted too, and it just disappears from the homepage, which I is probably what a user would expect anyway

Reply:
Alright! I've updated it.